certified-mail-sender/src/main/resources/logback.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
don't we use the log level splitter here?

Reply:
What do you mean?

Reply:
Ok I got the hint: you are talking about STD*

Reply:
correct, the different appenders to log to stdout or stderr based on the log level